shared/oval/sshd_limit_user_access.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
```

Comment:
Hi, we are in process of moving everything over there but it could be that it's not in git yet.

I guess in this case we can tolerate it being in the old location and move it later.